Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
Currently I define it in cmake (unconditionally). Android uses gnustl by default and it looks like there's no to_string (it is possible to use libc++, but it's not fully supported).
If there's a better way to detect this, I'm open to suggestions.